onnx/defs/experiments/functions.cc

```diff
@@ -175,17 +25,35 @@ ONNX_OPERATOR_SET_SCHEMA(
         .Output(0, "Y", "Output tensor", "T")
         .Attr(
             "axes",
-			"A list of integers, along which to reduce. The default is to reduce over "
-			"all the dimensions of the input tensor. Use [0,2,3] (without C axis for "
+            "A list of integers, along which to reduce. The default is to reduce over "
+            "all the dimensions of the input tensor. Use [0,2,3] (without C axis for "
             "N-D cases) for calculating means and variances along channels. Two "
             "variables with the same C-coordinate are associated "
             "with the same mean and variance.",
-			AttributeProto::INTS,
-			OPTIONAL)
+            AttributeProto::INTS,
+            OPTIONAL)
         .TypeConstraint(
             "T",
             {"tensor(float16)", "tensor(float)", "tensor(double)"},
             "Constrain input and output types to all numeric tensors.")
-       .FunctionBody(BuildMVNFunctionBody()));
+        .FunctionBody(FunctionBodyHelper::BuildNodes(
+            {// nodes: {outputs, op, inputs, attributes}
```

Comment:
An example to illustrate might be helpful: e.g., "{ {Z}, "Add", {"X", "Y"} }" represents "Z = Add(X,Y)".